whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestApiTest.java

```diff
@@ -41,14 +41,14 @@ public void getName() {
 
     @Test
     public void response_header_for_any_request() {
-        when(dateTimeProvider.getCurrentDateTime()).thenReturn(LocalDateTime.of(2013, 3, 3, 12, 55));
+        when(dateTimeProvider.getLocalDateTimeUtc()).thenReturn(LocalDateTime.of(2013, 3, 3, 12, 55));
 
         assertThat(subject.getResponseHeader(), is(" - From-Host: 127.0.0.1\n - Date/Time: Sun Mar 3 12:55:00 2013\n"));
     }
 
     @Test
     public void notification_header_for_any_request() {
-        when(dateTimeProvider.getCurrentDateTime()).thenReturn(LocalDateTime.of(2013, 3, 3, 12, 55));
+        when(dateTimeProvider.getLocalDateTimeUtc()).thenReturn(LocalDateTime.of(2013, 3, 3, 12, 55));
 
         assertThat(subject.getNotificationHeader(), is(" - From-Host: 127.0.0.1\n - Date/Time: Sun Mar 3 12:55:00 2013\n"));
```

Comment:
Not part of your changes but shouldn't we add the TZ to the date/time reported by whois

Reply:
We should. I'll update to include this

Reply:
We need the timezone in all timestamps (UTC = 'Z')

Reply:
Updated to include the 'Z'

Reply:
Thank you